mava/components/jax/building/environments.py

```diff
@@ -67,6 +68,7 @@ def config_class() -> Optional[Callable]:
 @dataclass
 class ExecutorEnvironmentLoopConfig:
     should_update: bool = True
+    executor_stats_wrapper_class: Any = DetailedPerAgentStatistics
```

Comment:
Should this not be an `Optional[DetailedPerAgentStatistics]` or it's base class?

Reply:
I think it is good to have the default here since without it I don't think we track the trainer stats etc. Or are you mostly referring to the type that we should change from `Any`? For that, I agree, the base class could be more appropriate. In general, going forward, I think we should do a bit of a rethink regarding these stats wrappers and find a cleaner way using the redesign instead of passing in wrapper classes in the config.

Reply:
I updated the type @sash-a @arnupretorius . I agree with redesigning the wrappers in future.

Reply:
I was refering to the type to change from `Any`. But I think the current solution is good until we think of a cleaner way.